src/components/EasterEgg/EasterEgg.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useState, useCallback } from 'react'
+import { useState, useCallback, memo } from 'react'
 import { FallingBunnies, FallingBunniesProps, useKonamiCheatCode } from '@pancakeswap/uikit'
 
 const EasterEgg: React.FC<FallingBunniesProps> = (props) => {
```

Comment:
I didn't know we can do this 🤯 maybe types import globally

Reply:
React namespace is imported thats why we can use without import